wire-ios-request-strategy/Sources/Request Strategies/Availability/AvailabilityRequestStrategy.swift

```diff
@@ -141,6 +141,13 @@ extension AvailabilityRequestStrategy: OTREntity {
         return nil
     }
 
+    public var expirationReasonCode: NSNumber? {
+        get {
+            return nil
+        }
+        set {}
+    }
+
```

Comment:
This differs from the other strategies? Why?

Reply:
Because the `AvailabilityRequestStrategy` extension conforms to the `OTREntity` protocol, and extensions cannot contain stored properties. In all other strategies, we conform the `OTREntity` protocol not in the extension.